### ISSUE TITLE
 Uncaught ReferenceError: require is not defined

### ISSUE THREAD

**Issue by yangshengqian0810** [MeForma/vue-toaster#12]
```
Uncaught ReferenceError: require is not defined
    at Proxy.mounted (vue-toaster.js:161)
    at callWithErrorHandling (runtime-dom.esm-bundler-d92ceebf.js:1292)
    at callWithAsyncErrorHandling (runtime-dom.esm-bundler-d92ceebf.js:1301)
    at Array.hook.__weh.hook.__weh (runtime-dom.esm-bundler-d92ceebf.js:3011)
    at flushPostFlushCbs (runtime-dom.esm-bundler-d92ceebf.js:1471)
    at Object.render (runtime-dom.esm-bundler-d92ceebf.js:5774)
    at render (runtime-dom.esm-bundler-d92ceebf.js:9175)
    at mount (vue-toaster.js:290)
    at Object.show (vue-toaster.js:307)
    at Object.success (vue-toaster.js:317)
```

**Comment by jprodrigues70** [comment]
Hi, @ysq352358871! Thanks for the issue report. Can you share your vue-toaster options and a step sequence to reproduce this error?

**Comment by jprodrigues70** [comment]
issue closed for lack of explanation

**Comment by Androz2091** [comment]
I'm getting the same error

**Comment by Androz2091** [comment]
Error line:
```js
mounted() {
    this.showNotice()

    if (this.useDefaultCss) {
      require('./themes/default/index.styl')
    }

    eventBus.$on('toast-clear', this.close)
  },
```

Error data
```js
vue.js:1184 [Vue warn]: Unhandled error during execution of mounted hook 
  at <Toast message="Logged as WHMCS BOT!" >
warn @ vue.js:1184
logError @ vue.js:1357
handleError @ vue.js:1349
callWithErrorHandling @ vue.js:1303
callWithAsyncErrorHandling @ vue.js:1309
hook.__weh.hook.__weh @ vue.js:3056
flushPostFlushCbs @ vue.js:1479
render @ vue.js:5942
render @ vue.js:9304
mount @ mount-component.js:14
show @ api.js:9
(anonymous) @ Settings.vue:109
Promise.then (async)
(anonymous) @ Settings.vue:104
Promise.then (async)
onSubmit @ Settings.vue:100
(anonymous) @ Settings.vue:4
(anonymous) @ vue.js:9216
callWithErrorHandling @ vue.js:1300
callWithAsyncErrorHandling @ vue.js:1309
invoker @ vue.js:8406

Toaster.vue:92 Uncaught (in promise) ReferenceError: require is not defined
    at Proxy.mounted (Toaster.vue:92)
    at callWithErrorHandling (vue.js:1300)
    at callWithAsyncErrorHandling (vue.js:1309)
    at Array.hook.__weh.hook.__weh (vue.js:3056)
    at flushPostFlushCbs (vue.js:1479)
    at Object.render (vue.js:5942)
    at render (vue.js:9304)
    at mount (mount-component.js:14)
    at Object.show (api.js:9)
    at Settings.vue:109
```

**Comment by Androz2091** [comment]
waiting for a fix :grin:
https://github.com/Androz2091/slash-commands-gui/blob/main/src/views/Settings.vue#L144

**Comment by jprodrigues70** [comment]
Fix in 1.2.1. Require was removed
